sdk/core/Azure.Core/src/AzureSasCredential.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.ComponentModel;
+using System.Threading;
+using Azure.Core;
+
+namespace Azure
+{
+    /// <summary>
+    /// Shared access signature credential used to authenticate to an Azure Service.
+    /// It provides the ability to update the signature without creating a new client.
+    /// </summary>
+    public class AzureSasCredential
+    {
+        private string _signature;
+
+        /// <summary>
+        /// Signature used to authenticate to an Azure service.
```

Comment:
Nit - let's use "Shared access signature" in all the doc comments.  The property/param names are fine though.